tests/cargo-kani/vecdeque-cve/Cargo.toml

```diff
@@ -8,8 +8,3 @@ description = "Reproduces CVE-2018-1000657"
 
 [lib]
 path = "src/harness.rs"
-
-[dependencies]
-
-[package.metadata.kani.flags]
-cbmc-args = ["--no-built-in-assertions"]
```

Comment:
I found this wasn't needed to verify the harnesses.

Reply:
@celinval Was there a reason why we removed these checks in the initial commit?

Reply:
I was trying to reduce the size of the report to make it more demo friendly. :)